### PR TITLE
[MINI-4198] Add Custom Permission `Attachments`

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Add Custom Permission `Attachments` and add optional permission to the manifest.
+ Update Readme with missing points permission + attachments

## Links
[https://jira.rakuten-it.com/jira/browse/MINI-4198](https://jira.rakuten-it.com/jira/browse/MINI-4198)

# Checklist
- [ ] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by Climbatize — APPROVED** [review]
LGTM

File: js-miniapp-bridge/src/types/custom-permissions.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export enum CustomPermissionName {
   LOCATION = 'rakuten.miniapp.device.LOCATION',
   SEND_MESSAGE = 'rakuten.miniapp.user.action.SEND_MESSAGE',
   POINTS = 'rakuten.miniapp.user.POINTS',
+  ATTACHMENTS = 'rakuten.miniapp.device.FILE_DOWNLOAD',
```
Comment (corycaywood) [inline js-miniapp-bridge/src/types/custom-permissions.ts:9]: I think we should also change the name to match:
```suggestion
  FILE_DOWNLOAD = 'rakuten.miniapp.device.FILE_DOWNLOAD',
```

Comment (laubengaier) [reply js-miniapp-bridge/src/types/custom-permissions.ts]: sure, fixed that 👍

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

**Pull request merged**